src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -114,7 +114,7 @@ private void InitializeCore()
                 return;
             }
 
-            var vsShell = _serviceProvider.GetService<SVsShell, IVsShell>();
+            var vsShell = IServiceProviderExtensions.GetService<SVsShell, IVsShell>(_serviceProvider);
```

Comment:
What happened here? (Ditto rest of file.)

Reply:
tehy collide now with an extension that comes in from VS.  i didn't want to cahnge semantics, so i explicitly made us call our extensino.  we can likely remove.  but i think that should be in a different PR.